docs/release-notes/4257-which.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Fixes**
+
+-  Supports running in containers which do not have a /bin/which path, such as python-slim. The
+   error was caused by accidentally accidentally hardcoding ``/bin/which`` instead of letting the
```

Comment:
accidentally 2 instances of "accidentally"

Reply:
doh